docs/continuing_runs.md

```diff
@@ -0,0 +1,204 @@
+# Continuing and editing runs
+
+The modularity of HADDOCK3 allows users to take parts of a run and use them as
+starting points for new runs using different configuration files. There are
+several ways users can continue or expand previous runs. The index below lists
+those possibilities:
+
+1. Restart a previous run from a given step
+1. Expanding a previous run from a given step
+1. Editing and expanding a previous run from a given step
+1. Copy a run to a new folder
+1. Extend an unknown run
```

Comment:
What is an unknown run???

Reply:
So you mean a run for which the config file is not present?  Which will kind of never happen since the config file is copied  in the run dir, right?  You have to have some knowledge of what is in a run in order to be able to setup the next steps.

This seems to me an unlikely scenario

Reply:
No, no. The config is surely there. And as users (human beings), we know what the run looks like and all the parameters. But at the moment of running an extension, the command line itself:

`haddock3 new-extension.cfg --restart <folder>`

would not know about the previous run parameters because these are not defined in `new-extension.cfg`, which would automatically be extracted from the run instead. Sometimes I overuse certain words, sorry. Is it clear the difference between:

`haddock3 new-extension.cfg --restart <folder>`

and

`haddock3 new-extension.cfg --restart <number>`

Those are the two option I wanted to implement in these docs.